auth-server-open-api-spec.yaml

```diff
@@ -516,6 +518,27 @@ components:
         - $ref: '#/components/schemas/access-incoming'
         - $ref: '#/components/schemas/access-outgoing'
       description: The access associated with the access token is described using objects that each contain multiple dimensions of access.
+    limits-outgoing-recurring:
+      title: limits-outgoing-recurring
+      description: Open Payments specific property that defines the limits under which outgoing payments can be created.
+      type: object
+      properties:
+        receiver:
+          type: string
+          format: uri
+          description: URL for the incoming payment or ILP STREAM Connection.
+        sendAmount:
+          $ref: '#/components/schemas/amount'
+        receiveAmount:
+          $ref: '#/components/schemas/amount'
+        interval:
+          type: string
+          description: '[ISO8601 repeating interval](https://en.wikipedia.org/wiki/ISO_8601#Repeating_intervals)'  
+      required: 
+        -  interval
+        -  oneOf
+          - sendAmount
+          - receiveAmount
```

Comment:
Does this allow both amounts?
https://stackoverflow.com/questions/31839578/how-to-define-a-json-schema-that-requires-at-least-one-of-many-properties/31841897#31841897

Reply:
Do we allow both amounts?